helix-query.yaml

```diff
@@ -8,7 +8,7 @@ indices:
       - en/publish/*/*/*/*.(md|docx)
     exclude:
       - '**/Document.*'
-    target: https://adobe.sharepoint.com/:x:/r/sites/TheBlog/Shared%20Documents/theblog/en/query-index.xlsx?d=we7bf6b3af3234076968b30a1565f2373&csf=1&web=1&e=q9o8tW
+    target: https://adobe.sharepoint.com/:x:/r/sites/TheBlog/Shared%20Documents/theblog/en/query-index.xlsx?d=w9b11cf8528564434a5363b1c13699d0f&csf=1&web=1&e=crkaXh
```

Comment:
Why has this changed?

Reply:
This was a direct commit to `master` which did not make it to `staging`.

Reply:
Ah, I think I know: @dominique-pfister switched to a new `query-index.xlsx` when he activated the new indexer.

Reply:
Yes, apologies: I replaced the old `en/query-index.xlsx` that had empty rows with a gapless one, that had identical contents.

Reply:
No worries at all! I was just wondering why this change would end up in this PR 🙂  all clear now!